CONTRIBUTING.md

```diff
@@ -18,31 +50,70 @@ key, but non-obvious, aspects:
 
 - All attributes, metrics, etc. are formally defined in YAML files under
   the `model/` directory.
-- All descriptions, normative language are defined in the `docs/`
-  directory.
-  - We provide tooling to generate Markdown documentation from the formal
```

Comment:
Why remove these bullet points?  Should we instead have links to the other sections?

Reply:
I created actual sections for these points I removed, because I think they were more or less "burried" in there and hard to find. Now they have their own place - Changes in YAML, Changes in markdown, Hugo headings, schema files and the changelog. 

I think this part if more useful for general things and not specific to "how to do it".